src/actions.js

```diff
@@ -10,6 +10,8 @@ import {
   formatServerError,
 } from "./helpers/api";
 
+const REQUESTED_WITH = 'webapp'
```

Comment:
new constant for web application source (something simillar must be done on Android app level I presume @delcroip )